polkit/authority.go

```diff
@@ -75,6 +75,7 @@ func CheckAuthorizationForPid(pid uint32, actionId string, details map[string]st
 		return false, err
 	}
 	subject.Details["start-time"] = dbus.MakeVariant(startTime)
```

Comment:
Not sure why we are setting the start-time like this. We could just pass '0' to polkit and it would do the /proc lookup itself. Either way, it would be good for future maintenance to add a comment along the lines of:
```
// While discovering the pid's start time is racy, it isn't security
// relevant since it only impacts expiring the permission after
// process exit.
```